enterprise/frontend/src/metabase-enterprise/embedding_iframe_sdk/embed.v1.unit.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { MetabaseEmbed } from "./embed.v1";
+
+describe("embed.v1.js script tag for sdk iframe embedding", () => {
+  const defaultSettings = {
+    apiKey: "test-api-key",
+    instanceUrl: "http://localhost:3000",
+    target: "#embed-container",
```

Comment:
This gives me the illusion that it will pass without throwing embed container not found error if I use this ID, bet that's not the case. Whatever ID we pass here would fail. It's just that this error is thrown after other errors.

Maybe we could just set `#non-existent-target` here. What do you think?

Reply:
Fixed in https://github.com/metabase/metabase/pull/57269/commits/a311c4caae6a4dfc5d01c857766f3828d6a67b6f, thanks!